.github/workflows/publish-draft-releases.yml

```diff
@@ -921,20 +1312,20 @@ jobs:
           github-token: ${{ secrets.GH_SHR_TOKEN }}
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-instance-ssh-public-key: ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIPb24HEQ++aNFWaqVyMtIs6GotUB8R+q61XOoI2z6uMj
-          aws-region: us-east-1
-          aws-subnet-id: subnet-08c26caf0a52b7c19
-          aws-security-group-id: sg-0315bffea9042ac9b
+          # aws-instance-ssh-public-key: ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIPb24HEQ++aNFWaqVyMtIs6GotUB8R+q61XOoI2z6uMj
+          aws-region: us-west-1
+          # aws-subnet-id: subnet-08c26caf0a52b7c19
+          # aws-security-group-id: sg-0315bffea9042ac9b
```

Comment:
This is a workaround for an AWS error MaxSpotInstanceCountExceeded: Max spot instance count exceeded

TODO: revert before merging.